packages/wrangler/e2e/unenv-preset/worker/index.ts

```diff
@@ -69,6 +69,16 @@ export const WorkerdTests: Record<string, () => void> = {
 		assert(array.every((v) => v >= 0 && v <= 0xff_ff_ff_ff));
 	},
 
+	async testCrypto() {
+		const crypto = await import("node:crypto");
+
+		assert.strictEqual(typeof crypto.pseudoRandomBytes, "function");
+		assert.strictEqual(typeof crypto.Cipher, "function");
+		assert.strictEqual(typeof crypto.Decipher, "function");
+		assert.strictEqual(typeof crypto.createCipher, "function");
+		assert.strictEqual(typeof crypto.createDecipher, "function");
```

Comment:
Could you please update this tests to work with either value of [`remove_nodejs_compat_eol_v22'](https://github.com/cloudflare/workerd/blob/94fa1595fb86b8989242005be59c505cf6f29b8e/src/node/crypto.ts#L541-L559))

```ts
		const removeEolV22 = getRuntimeFlagValue(
			"remove_nodejs_compat_eol_v22"
		);

    if (removeEolV22) {
      assert.strictEqual(crypto.Cipher, undefined);
      // ...
    } else {
      assert.strictEqual(typeof crypto.Cipher, "function");
      // ...
    }      
```